docs/methoddocs/alerts.md

```diff
@@ -1,8 +1,16 @@
-# Alerts
```

Comment:
Other reviewers: Note that this file was re-ordered to handle a docs build issue that arose from importing the `AlertState` constants